exercise-analyst-server/API/Services/Auth/Dtos/LoginResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace API.Services.Auth.Dtos
+{
+    public class LoginResponse
+    {
+        public string AccessToken { get; set; }
```

Comment:
Może coś mi się pomieszało ale chyba z samego tokenu frontend się nie ogarnie jakie ma role etc i będzie ciężko z interface'm

Reply:
W sumie to da sie zdekodować token na froncie i powyciągać role itd, ale fakt - łatwiej będzie po prostu dostać je z backendu jako osobne pola :D 

Dodałem role, email, wzrost, wage, wiek, płeć i date utworzenia konta oprócz samego tokena. Wystarczy? Albo coś wywalić? :D